frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -226,12 +228,20 @@ const setup = async ({
     setupModelIndexEndpoints(card.id, []);
   }
 
+  // this workaround can be removed when metabase#34523 is fixed
+  if (card === null) {
+    fetchMock.get("path:/api/model-index", [createMockModelIndex()]);
+  }
+
   const mockEventListener = jest.spyOn(window, "addEventListener");
 
   const { history } = renderWithProviders(
     <Route>
       <Route path="/home" component={TestHome} />
       <Route path="/model">
+        <Route path="new" component={NewModelOptions} />
```

Comment:
We could do without this route, but it makes the test more "real". No need to mock URLs of new notebook model or new native model.